pkg/virt-config/feature-gates.go

```diff
@@ -34,6 +34,11 @@ const (
 	SnapshotGate          = "Snapshot"
 	HostDiskGate          = "HostDisk"
 	VirtIOFSGate          = "ExperimentalVirtiofsSupport"
+	// 	CPUMigrationGate allows users to block migrations for virtual machines with host-model
+	CPUMigrationGate = "MigrationCPU"
```

Comment:
this name is too opaque for me - I cannot tell what it means. How about `MigratableHostModel` ?

Reply:
@dankenigsberg Completely agree. I haven't started the re-factoring. It was only the rebase. Now I will update the PR code itself according to the discussion